javascript/javascript2/week3/script.js

```diff
@@ -0,0 +1,130 @@
+// 1.Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+const delayTime = () => {
+    setTimeout(() => { console.log('Called after 2.5 seconds'); }, 2500);
+}
+window.addEventListener('load', delayTime);
+
+// 2. Create a function that takes 2 parameters: delay and stringToLog.
+// Calling this function should log out the stringToLog after delay seconds.
+// Call the function you have created with some different arguments.
+
+function stringAfterDelay(delay, stringToLog) {
+    setTimeout(() => { console.log(`${stringToLog}`); }, delay * 1000);
+}
+
+stringAfterDelay(2, 'This string is running after 2 seconds');
+stringAfterDelay(5, 'This string is running after 5 seconds');
+
+// 3. Create a button in html. When clicking this button, use the function you created in the previous task
+// to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+
```

Comment:
If you want to get fancy, you could replace every use of document.getElementById(id) with 
```suggestion
let $ = (id) => { return document.getElementById(id) }
const delayButton = $('button')
```